src/foam/util/LRULinkedHashMap.java

```diff
@@ -0,0 +1,20 @@
+package foam.nanos.util;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/**
+ * Created by marcroopchand on 2017-05-24.
+ */
+public class LRULinkedHashMap<K, V> extends LinkedHashMap<K, V> {
+  private final int MAX_SIZE;
```

Comment:
Rename MAX_SIZE to maxSize_. CONSTANT naming convention implies it's a constant, but it's just final.